cli/cmd/install/install.go

```diff
@@ -32,7 +31,7 @@ func init() {
 }
 
 func addInstallUtilitiesFlags(cmd *cobra.Command) {
-	cmd.Flags().BoolP("only-secrets", "", false, "Option to create only TimescaleDB secrets")
+	cmd.Flags().BoolP("only-secrets", "", false, "[DEPRECATED] Option to create only TimescaleDB secrets")
```

Comment:
Should the description say that this is non-operational? Even if its deprecated (hence scheduled for removing) users could expect it to work.